content/get/index.md

```diff
@@ -1,265 +1,32 @@
-
-</div>
-</div>
-
-  <div class="tab" id="suse">
-     <input type="radio" id="tab-suse" name="tabs" >
-     <label for="tab-suse"><img src="/public/images/skin/rexify.org/suse.png"></label>
-
-     <div class="content">
-<h2>For 13.1</h2>
-
-<pre><code class="bash">    $ rpm --import https://rex.linux-files.org/RPM-GPG-KEY-REXIFY-REPO
-    $ zypper addrepo -t rpm-md -f -n rex https://rex.linux-files.org/OpenSuSE/13.1/rex/x86_64/ rex
-    $ zypper install rex
-</code></pre>
-
-<h2>For 13.2</h2>
-
-<pre><code class="bash">    $ rpm --import https://rex.linux-files.org/RPM-GPG-KEY-REXIFY-REPO
-    $ zypper addrepo -t rpm-md -f -n rex https://rex.linux-files.org/OpenSuSE/13.2/rex/x86_64/ rex
-    $ zypper install rex
-</code></pre>
-</div>
-</div>
-
-  <div class="tab" id="fedora">
-     <input type="radio" id="tab-fedora" name="tabs" >
-     <label for="tab-fedora"><img src="/public/images/skin/rexify.org/fedora.png"></label>
-
-     <div class="content">
-<h2>Fedora 20+</h2>
-
-<pre><code class="bash">    $ rpm --import https://rex.linux-files.org/RPM-GPG-KEY-REXIFY-REPO
-
-    $ cat &lt;/etc/yum.repos.d/rex.repo &gt;&gt;EOF
-    [rex]
-    name=Fedora \$releasever - \$basearch - Rex Repository
-    baseurl=https://rex.linux-files.org/Fedora/\$releasever/rex/\$basearch/
-    enabled=1
-    EOF
-
-    $ yum install rex
-</code></pre>
-
-</div>
-</div>
-
-  <div class="tab" id="freebsd">
-     <input type="radio" id="tab-freebsd" name="tabs" >
-     <label for="tab-freebsd"><img src="/public/images/skin/rexify.org/freebsd.png"></label>
-
-     <div class="content">
-<h2>Installation with ports</h2>
-
-Thanks to Sam Cassiba Rex is now available in the official FreeBSD Ports collection.
-
-<pre><code class="bash">    $ cd /usr/ports/sysutils/p5-Rex
-    $ make install clean
-</code></pre>
-
-<h2>Installation with pkg</h2>
-
-<pre><code class="bash">    $ pkg install p5-Rex
-</code></pre>
-
-</div>
-</div>
-
-</div>
-</div>
-<div style="clear:both"></div>
-
-## Apple Mac OS X
-
-To install Rex on Mac OSX you have to install libssh2 first. You can do this with MacPorts or with Homebrew.
-
-We recommend using MacPorts. First you need to install XCode. After you've installed it you'll find a package for your system here.
+## CPAN
 
-If you're using homebrew we recommend to install a custom perl with Perlbrew.
+The recommended way to get Rex is to install it from CPAN. You can simply use your preferred client, for example:
 
-Installation instructions for MacPorts:
+    $ cpanm Rex
 
-    $ sudo port install libssh2 perl5
-    $ curl -L https://get.rexify.org | perl - --sudo -n Rex
+## Package managers
 
-## Microsoft Windows
+### Official packages
 
-Tested with Windows Vista and Windows 7 (64bit) and Strawberry Perl 5.x
+Rex is distributed via package managers for many different operating systems. The [Repology page of Rex](https://repology.org/project/rex/versions) is perhaps the most complete and up-to-date list of available Rex packages.
```

Comment:
Nice find! I think it's a really good idea to depend on an external source like that!

Reply:
Yes, I rediscovered the site just in time for this PR :) I'm glad you both like this approach!